packages/api/README.md

```diff
@@ -1,131 +1,107 @@
-# Sabia API
+# Sabiá API
 
-## Como executar o servidor da API?
+## Table of Contents
 
-1. Crie um arquivo `.env`, baseando-se no arquivo `.env.example`;
-2. Crie um banco chamado `sabia` (mesmo nome da key `DB_DATABASE` dentro do arquivo `.env`);
-3. Execute as migrations para criar as tabelas no banco:
+- [Running the API Server](#running-the-api-server)
+- [Running the Tests](#running-the-tests)
+- [API Documentation](#api-documentation)
+- [Using Fake Values](#using-fake-values)
+- [Pushing the Data to Algolia](#pushing-the-data-to-algolia)
+- [Sending Emails](#sending-emails)
+- [Uploading Files](#uploading-files)
+- [Adonis Framework](#adonis-framework)
 
+## Running the API Server
+
+1. Rename the `.env.example` file to `.env` and update all of the environment variables correctly.
+2. Make sure you have a MySQL database server (feel free to use docker or whatever method you prefer) up and running and update the `DB_HOST`, `DB_PORT`, `DB_USER` and `DB_PASSWORD` environment variables.
```

Comment:
Seria bom informar que é o MySQL 5.7 já que a versão mais nova não funciona no docker atualmente por problemas no driver de autenticação.

Caso queira deixar mais completo é possível adicionar uma instrução para executar esse banco pelo docker:
`docker run --name sabia_mysql -e MYSQL_ROOT_PASSWORD=your_password -v /your/own/datadir:/var/lib/mysql -p 3306:3306 mysql:5.7`

Reply:
Oi @calaca , eu posso colocar essa versão sim :+1: . Mas antes de eu colocar deixa eu só entender melhor: quando você testou, viu que a versão mais nova não funciona no docker ou não funciona com o nosso projeto em si? Porque se for no docker, não seria um problema para o nosso projeto, entende? Porque o docker é só uma das formas que o usuário vai poder rodar o banco. Ele pode usar um servidor externo ou mesmo configurar direto na máquina, por exemplo. Eu acho que essa parte de configuração usando o docker devemos deixar fora, pra deixar o mais limpo possível. Se ele precisar, vai buscar como subir o banco.

O que acha?

Reply:
@lcnogueira confesso que não testei fora do docker pois não gosto de instalar banco na minha máquina. Mas caso ache que não cabe essa informação na documentação, tudo bem! Realmente é uma particularidade da imagem "quebrada" do MySQL mais recente.

Reply:
É, acho que é isso mesmo: mais uma particularidade da questão da imagem do docker em si. Acho que, por hora, deixemos essa informação de fora. Caso percebamos mais na frente com algum usuário, gente atualiza aqui :+1: